src/store/index.ts

```diff
@@ -0,0 +1,31 @@
+import { applyMiddleware, compose, createStore } from "redux";
+import createSagaMiddleware from "redux-saga";
+import createReducer from "./reducer";
+
+
+declare global {
+  interface Window {
+    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
+  }
+}
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Since these have to do with redux dev tools which is only used in development, my gut is telling me that this should not be checked into and used in production code even though it will fall back on `compose`. Perhaps it's better to use https://github.com/zalmoxisus/redux-devtools-extension instead and use `redux-devtools-extension/developmentOnly` which is mentioned [here](https://github.com/zalmoxisus/redux-devtools-extension#14-using-in-production).